ProjectApplication/project_core/filters.py

```diff
@@ -6,22 +6,24 @@
 
 
 class ProjectFilterSet(FilterSet):
-    key_title = CharFilter(method='filter_key_title', label="Key or Title")
-    keywords = CharFilter(method='filter_keywords')
+    key_title = CharFilter(method='filter_any', label="Key, Title, Keyword, PI, laysummary or organisation")
```

Comment:
`key_title` might need to be changed to something else? (to reflect that is a "filter_any" not "filter_key_title" as before)